packages/components/src/__future__/Tag/Tag.module.scss

```diff
@@ -0,0 +1,66 @@
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/typography";
+
+.container {
+  display: inline-flex;
+  align-items: center;
+  padding: 5px 0.5rem 5px 0.5rem;
+  gap: 0.25rem;
```

Comment:
It's not a good idea to mix measurements, because they won't scale properly.

Use the spacing tokens where possible, and then use a rem equivalent of any px value if it doesn't exist as a token.